rotkehlchen/chain/gnosis/modules/airdrops/decoder.py

```diff
@@ -1,36 +0,0 @@
-import logging
```

Comment:
Hmm so it seems this was an empty decoder class? Right?

Reply:
yes, the only thing it was doing was to add the cowswap counterparty but we already have it with the decoder for the swaps. This one was a no-op it seems